dali/dfu/dfurun.cpp

```diff
@@ -595,12 +595,50 @@ class CDFUengine: public CInterface, implements IDFUengine
             auditflags |= DALI_LDAP_WRITE_WANTED;
         SecAccessFlags perm = queryDistributedFileDirectory().getFDescPermissions(fd,user,auditflags);
         IDFS_Exception *e = NULL;
-        if (!HASREADPERMISSION(perm))
+        if (!write&&!HASREADPERMISSION(perm))
             throw MakeStringException(DFSERR_LookupAccessDenied,"Lookup permission denied for physical file(s)");
```

Comment:
trivial: but worth changing these to makeStringExceptionV while here (include filename)

Reply:
What if there are multiple files in the fd? Should I use fd->getTraceName() to get a name (filename?) here?

Reply:
I guess I can use the name returned by fd->getTraceName() which returns all file names if multiple.

Reply:
might be big..
maybe spot the multi case, and truncate if >255 with "..."